README.md

```diff
@@ -77,6 +110,10 @@ Options will be passed through to [HTTPoison](https://github.com/edgurgel/httpoi
 Closex.HTTPClient.get_lead("my_lead_id", timeout: 500, recv_timeout: 1_000)
 ```
 
+## Mock Client
+
+We have provided a mock client for testing purposes in your application.
```

Comment:
? any examples

Reply:
Usage, etc was in the mock client docs but I've moved them over to here now. Mock client docs now mostly contain detail around method usage and json file overriding.